<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CTrainManagerTable</class>
 <widget class="QDialog" name="CTrainManagerTable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>230</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>学员培训管理</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 14pt &quot;宋体&quot;;</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>模 式：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboSimuMode">
       <item>
        <property name="text">
         <string>培训模式</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>考试模式</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>竞赛模式</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>闭锁控制：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_interlock">
       <item>
        <property name="text">
         <string>闭锁并提示</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>闭锁不提示</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>无闭锁，任意操作</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>仿真速度：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelSimuSpeed">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <widget class="QSlider" name="sliderSimuSpeed">
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>7</number>
         </property>
         <property name="pageStep">
          <number>1</number>
         </property>
         <property name="value">
          <number>1</number>
         </property>
         <property name="sliderPosition">
          <number>1</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="invertedAppearance">
          <bool>false</bool>
         </property>
         <property name="invertedControls">
          <bool>false</bool>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksBothSides</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="buttonSimuStart">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>开始仿真</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonSimuPause">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>暂停仿真</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonSimuStop">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>停止仿真</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
